rust/connlib/clients/android/connlib/build.gradle.kts

```diff
@@ -79,6 +82,36 @@ dependencies {
 
 apply(plugin = "org.mozilla.rust-android-gradle.rust-android")
 
+tasks.register("copyJniSharedObjects") {
+    outputs.upToDateWhen { false }
+
+    val jniTargets = mapOf(
+        "armv7-linux-androideabi" to "armeabi-v7a",
+        "aarch64-linux-android" to "arm64-v8a",
+        "i686-linux-android" to "x86",
+        "x86_64-linux-android" to "x86_64",
+    )
+
+    jniTargets.forEach { entry ->
+        val soFile = File(
+            project.projectDir.parentFile.parentFile.parentFile.parentFile,
+            "target/${entry.key}/debug/libconnlib.so"
```

Comment:
`debug` is only right for debug builds; for release builds this should be `release`